Vagrantfile

```diff
@@ -20,7 +22,9 @@ Vagrant.configure(2) do |config|
     puppet.manifests_path = "puppet"
     puppet.module_path = "puppet/modules"
     puppet.facter = {
-    	"quickstart_domain" => config.vm.hostname
+        "hostname"          => config.vm.hostname,
```

Comment:
Good catch. I should've actually switched to the `fqdn` fact, otherwise we don't end up with a TLD. Updates forthcoming.